ni_measurementlink_service/_internal/discovery_client.py

```diff
@@ -260,11 +260,13 @@ def _key_file_exists(key_file_path: pathlib.Path) -> bool:
 def _start_service(exe_file_path: pathlib.PurePath, key_file_path: pathlib.Path) -> None:
     """Starts the service at the specified path and wait for the service to get up and running."""
     global _discovery_service_subprocess  # save Popen object to avoid ResourceWarning
+    creationflags = subprocess.CREATE_BREAKAWAY_FROM_JOB
```

Comment:
Why do we need a variable for one usage?

Reply:
I have reverted back. For test purpose I made the change since `Mypy` was throwing error on the `CREATE_BREAKAWAY_FROM_JOB`. Now I have used the hexadecimal equivalent `0x1000000` to resolve the `Mypy` issue